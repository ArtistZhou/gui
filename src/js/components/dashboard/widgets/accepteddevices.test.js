import React from 'react';
import renderer from 'react-test-renderer';
import AcceptedDevices from './accepteddevices';

describe('AcceptedDevices Component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<AcceptedDevices />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
