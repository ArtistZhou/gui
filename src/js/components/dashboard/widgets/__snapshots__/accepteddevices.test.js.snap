// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="notActive widget "
  onClick={[Function]}
>
  <div
    style={
      Object {
        "display": "flex",
        "flexDirection": "column",
        "height": "100%",
        "width": "100%",
      }
    }
  >
    <div
      className="widgetMainContent align-right"
      style={
        Object {
          "alignItems": "flex-end",
          "display": "flex",
          "flexDirection": "column",
        }
      }
    >
      <div
        className="header"
      >
        Accepted devices
      </div>
      <div
        className="counter"
      />
    </div>
    <span
      className="link"
    />
  </div>
</div>
`;
